### PR TITLE
Provide minor adjustments to the language of the chapter

### PR DESCRIPTION
**Pull request by m-czernek**

### CODE REVIEW

File: optaplanner-docs/src/main/docbook/en-US/Chapter-Design_patterns/Chapter-Design_patterns.xml [hunk 1]
```diff
@@ -228,34 +226,34 @@
...
+              loading or unloading time at a customer location) and also needs to rest 10 hours after 14 hours of
               work.</para>
             </listitem>
           </itemizedlist>
         </listitem>
 
         <listitem>
-          <para>Planning variable gaps: This is uncommon, because this extra planning variable (which impacts <link
-          linkend="searchSpaceSize">the search space</link>) reduces efficiency and scalability.</para>
+          <para>Planning variable gaps: This is uncommon, because such extra planning variable (which impacts the <link
```
Comment (ge0ffrey) [inline optaplanner-docs/src/main/docbook/en-US/Chapter-Design_patterns/Chapter-Design_patterns.xml]: "such extra planning variable" => might miss an "an"?

File: optaplanner-docs/src/main/docbook/en-US/Chapter-Design_patterns/Chapter-Design_patterns.xml [hunk 2]
```diff
@@ -228,34 +226,34 @@
...
     <para>Each stage has its own solver configuration (and therefore its own <literal>SolverFactory</literal>). Do not
-    confuse with <link linkend="solverPhase">multi phase solving</link> which uses only 1 solver configuration.</para>
+    confuse it with <link linkend="solverPhase">multi-phase solving</link> which uses a one-solver configuration.</para>
 
-    <para>Just like <link linkend="partitionedSearch">Partitioned Search</link>, multi stage planning leads to
-    suboptimal results. Nevertheless, in complex cases it's often worth it to simplify maintenance and ownership, as
-    well as to get the project off the ground.</para>
+    <para>Similarly to <link linkend="partitionedSearch">Partitioned Search</link>, multi-stage planning leads to
+    suboptimal results. Nevertheless, it may be beneficial in order to simplify the maintenance, ownership, and help a project
+    to succeed.</para>
```
Comment (ge0ffrey) [inline optaplanner-docs/src/main/docbook/en-US/Chapter-Design_patterns/Chapter-Design_patterns.xml]: It's not so much about getting the project to succeed, as it is to get the project started, get it out the gate, get it off the ground, basically avoid "analysis paralysis".

**Comment by ge0ffrey** [comment]
thanks for submitting a PR! I made 2 comments, please take a look at them: if you do any changed due to that, just do an extra commit on your existing branch and push it to your remote repository, this PR will automatically update. (1 branch == 1 PR)

**Comment by m-czernek** [comment]
Hi! Thank you for such a fast response. 

Ad 1) I find this wording awkward. 'Such' should count as a determiner, but it still requires additional a/an according to the Cambridge Dictionary. Hence I decided to change the wording and simplify it.

Ad 2) The whole sentence was very difficult to understand. I apologize for my confusion, it should be better now.

**Comment by ge0ffrey** [comment]
Thanks for fixing it. Your changes (= your commits) are good to merge now, but the PR changes have be unclear, click on the "files" tab here in this PR page. Some info about that:

1) To avoid that, it's generally better when syncing with upstream to rebase your work on top of master (so something like "git checkout mybranch; git fetch origin; git rebase origin/master") See https://www.atlassian.com/git/tutorials/merging-vs-rebasing/conceptual-overview

2) To make things easy, I 'll just cherry-pick your 2 commits from this PR into master and close this PR.

Thanks for sending this PR!

**Comment by ge0ffrey** [comment]
Manually merged by doing 2 cherry picks:
  http://github.com/droolsjbpm/optaplanner/compare/4cecb93...e7921a7

**Pull request closed without merging**